apps/src/feedbackBlocks.js

```diff
@@ -1,5 +1,5 @@
 var constants = require('./constants');
-var readonly = require('./templates/readonly.html');
+var readonly = require('./templates/readonly.html.ejs');
```

Comment:
yeah, I looked a bit but didn't see anything obvious. this change was needed because I've replace the ejs grunt task with an inline browserify transform. i figure this find/replace would be okay as a one-time change